File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopLinkJpaRepository.java
package com.cakk.domain.mysql.repository.jpa;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.mysql.entity.shop.CakeShopLink;

public interface CakeShopLinkJpaRepository extends JpaRepository<CakeShopLink, Long> {

	List<CakeShopLink> findAllByCakeShopId(Long cakeShopId);
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopLinkJpaRepository.kt
package com.cakk.domain.mysql.repository.jpa

import com.cakk.domain.mysql.entity.shop.CakeShopLink
import org.springframework.data.jpa.repository.JpaRepository

interface CakeShopLinkJpaRepository : JpaRepository<CakeShopLink, Long> {
    fun findAllByCakeShopId(cakeShopId: Long): List<CakeShopLink>
}
